### DISCUSSION TITLE
Why doesn’t it have a search function yet?

### DISCUSSION THREAD

**Discussion by L-J-Yinph** [saicaca/fuwari#236] [Ideas]
Why doesn’t it have a search function yet? It is recommended to update quickly

**Comment by 3kh0** [comment]
![image](https://github.com/user-attachments/assets/e35ab9dc-b5bd-4fa0-b860-4baa3a645e25)

What do you mean? There is a search bar at the top

**Reply by L-J-Yinph** [reply]
A……I mean, there's a search button, but why isn't there a search function, do you understand?

**Reply by saicaca** [reply]
> A……I mean, there's a search button, but why isn't there a search function, do you understand?

There might be some misunderstanding here. The search function is available and will display results after entering text into the searchbar. But:
- It won't work in the dev environment.
- The first search takes some time to finish, which might make it seem like it's not working. I'll try to fix it later.

**Reply by L-J-Yinph** [reply]
Oh sorry, I hadn't updated it back then. _**And**_ I also sincerely wish you a better project and a better experience for us users.
